Makefile

```diff
@@ -19,6 +19,8 @@ SOURCES     := $(shell find $(SOURCEDIRS) -name '*.go' -print)
 # End-to-end tests overall run timeout.
 TEST_E2E_TIMEOUT ?= 60m
 
+RESOURCE_CTRL ?= false
```

Comment:
I think default should be `true` for this.
Btw do you want to include peerpodconfig-ctrl as part of this PR ?

Reply:
The reason it's not true as it was tested only with aws,azure and libvirt
I can add peerpodconfig-ctrl, but can it be installed as standalone controller? jens @jensfr may know ?

Reply:
After discussing with Jens, there's some overlapping ops that are done both by `make deploy ` and peerpodconfig-ctrl, hence, it will be a bit more complex to ingrate it, i'll suggest to do it in separate if needed